docker-compose.yml

```diff
@@ -191,12 +191,16 @@ services:
     init: true
     cap_add:
     - NET_ADMIN
+    ports:
+      - "53:53"
```

Comment:
could be restricted to one or more explicitly given IP addresses

Reply:
desec-stack does not know the public-facing IP address